### DISCUSSION TITLE
Suricata Security Onion

### DISCUSSION THREAD

**Discussion by BoualiR** [Security-Onion-Solutions/securityonion#10133] [2.4]
hi , so i have problems i can't see any suricata alerts in security onion console i test with ping but nothing happen but you should know that just when i installed security onion on my ubuntu machine i could see it in Alerts menu and in Kibana .

**Comment by cm-ops** [comment]
Would you share more about your install? https://github.com/Security-Onion-Solutions/securityonion/discussions/1720
